homeassistant/helpers/config_validation.py

```diff
@@ -80,6 +80,17 @@ def isfile(value: Any) -> str:
     return file_in
 
 
+def isdevice(value: Any) -> str:
```

Comment:
Oh we already have an `isfile`. Any reason why we can't merge this ? Isn't link just a symlink, which means it can still point at another file too ?